src/app/loadout/plug-drawer/SelectablePlug.tsx

```diff
@@ -75,24 +131,23 @@ function SelectablePlugDetails({
   const stats = getPlugDefStats(plug, classType);
 
   // We don't show Clarity descriptions here due to layout concerns, see #9318 / #8641
-  const plugDescriptions = usePlugDescriptions(plug, stats, /* forceUseBungieDescriptions */ true);
+  const plugDescriptions = usePlugDescriptions(plug, stats, /* forceUseBungieDescriptions */ false);
```

Comment:
Having to put a comment like that for a boolean arg just signals it should be a named field in an object.

```suggestion
  const plugDescriptions = usePlugDescriptions({ plug, stats, forceUseBungieDescriptions: false });
```

Reply:
Fwiw I specifically added this argument for the mods picker and this is the only caller that passes this argument (and `false` is equivalent to `undefined`) so we could just get rid of it entirely now.